modules/sievefilters/site.js

```diff
@@ -863,7 +850,22 @@ $(function () {
             );
         });
 
-
+        /**
+         * Toggle Filter
+         */
+        $('.toggle_filter').on('change', function () {
+                Hm_Ajax.request(
+                    [   {'name': 'hm_ajax_hook', 'value': 'ajax_sieve_toggle_script_state'},
+                        {'name': 'imap_account', 'value': $(this).attr('imap_account')},
+                        {'name': 'script_state', 'value': $(this).prop('checked')},
+                        {'name': 'sieve_script_name', 'value': $(this).attr('script_name')}],
+                    function(res) {
+                        if (res.router_user_msgs[0].startsWith('ERR')) {
```

Comment:
Please return proper json to check if it is successful operation or not. Router error messages are meant to display notices and errors and might not contain the error in the first place.

Reply:
hello @kroky  I think I have made the fixes please review

Reply:
You still check the router messages. Instead, return a json status from the method and inspect it e.g. `res.status == 'OK'` or something similar. These errors are already displayed by the ajaxComplete or similar hooks of our ajax system, so you shouldn't alert them. See how other ajax methods deal with JSON data returned by the server.

Reply:
hello @kroky 
if (res.status !== 'OK' || res.success !== true) {
  checkbox.prop('checked', !checkbox.prop('checked'));
   }
   I added this condition for Status Check and Success in the JSON response

Reply:
That's good but I don't see how you return this status OK structure from an output module...